understanding/22/accessible-authentication-enhanced.html

```diff
@@ -50,13 +50,11 @@ <h2>Intent of Accessible Authentication (Enhanced)</h2>
 
         <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use, and secure method to log in, access content, and undertake tasks. This criterion is the same as <a href="accessible-authentication">Accessible Authentication</a> but without the exceptions for objects and user-provided content.</p>
 
-        <p>The scenarios where the two exceptions might apply are authentication mechanisms which:</p>
+        <p> Any step of the authentication process:</p>
         <ul>
-            <li>display a selection of images, and the user must choose which image they provided;</li>
-            <li>display a selection of items as text, and the user must choose which they had provided;</li>
-            <li>display a selection of images, and the user must choose the images which contain an object such as a car.</li>
+            <li>cannot display a selection of images, videos, or audio clips, where users must choose which image they provided;</li>
```

Comment:
I agree that the original wording is a bit confusing, but I think this new wording is still problematic; as I understand it, a step of an authentication process **can** display the stuff in these two bullet points **if** it provides an alternative or mechanism.

I think the quickest fix would be to change L53 to something like `Any **required** step of the authentication process without an assistance mechanism:`. Ideally, I'd like to see the understanding doc provide a more details understanding about what exactly qualifies as "a mechanism... to assist the user in completing the cognitive function test", but that should probably be its own PR